src/SendGrid/Helpers/Mail/Model/Personalization.cs

```diff
@@ -20,52 +21,53 @@ public class Personalization
         /// </summary>
         [JsonProperty(PropertyName = "to", IsReference = false)]
         [JsonConverter(typeof(RemoveDuplicatesConverter<EmailAddress>))]
-        public List<EmailAddress> Tos { get; set; }
+        public List<EmailAddress>? Tos { get; set; }
```

Comment:
At least one `To` object is required for personalizations, so I'm not sure that this should be nullable ([reference](https://docs.sendgrid.com/for-developers/sending-email/personalizations)).